sql/core/src/test/resources/sql-tests/results/ansi/interval.sql.out

```diff
@@ -1213,7 +1213,7 @@ struct<>
 -- !query output
 org.apache.spark.sql.catalyst.parser.ParseException
 
-Intervals FROM year TO second are not supported.(line 1, pos 16)
+The feature is not supported: Intervals FROM year TO second are not supported(line 1, pos 16)
```

Comment:
It would be better to put a point at the end:
not supported(line -> not supported.(line

Reply:
Yes included